src/nspawn/nspawn-cgroup.c

```diff
@@ -5,14 +5,19 @@
 #include "alloc-util.h"
 #include "fd-util.h"
 #include "fileio.h"
+#include "fs-util.h"
 #include "mkdir.h"
 #include "mount-util.h"
-#include "nspawn-cgroup.h"
+#include "path-util.h"
 #include "rm-rf.h"
 #include "string-util.h"
 #include "strv.h"
+#include "user-util.h"
 #include "util.h"
 
+#include "nspawn-cgroup.h"
+#include "nspawn-mount.h"
```

Comment:
we don't separate these out usually. Only our public installed headers (i.e. those with an "sd-" prefix) are separated out